lib/primer/view_components/linters/blankslate_component_migration_counter.rb

```diff
@@ -6,7 +6,7 @@ module ERBLint
   module Linters
     # Counts the number of times a HTML Blankslate is used instead of the component.
     class BlankslateComponentMigrationCounter < BaseLinter
-      MESSAGE = "We are migrating Blankslate to use [Primer::Beta::Blankslate](https://primer.style/view-components/components/beta/blankslate), please try to use that instead of raw HTML."
+      MESSAGE = "We are migrating Blankslate to use [Primer::Beta::Blankslate](https://primer.style/components/blankslate/rails/beta), please try to use that instead of raw HTML."
```

Comment:
This is not strictly part of this PR, but I noticed the link was wrong so I decided to fix it.